server/src/KleeRunner.cpp

```diff
@@ -129,6 +129,15 @@ void KleeRunner::processBatchWithoutInteractive(MethodKtests &ktestChunk,
     if (!tests.isFilePresentedInArtifact) {
         return;
     }
+    if (testMethod.sourceFilePath != tests.sourceFilePath) {
+        std::stringstream message;
+        message << "While generating tests for source file: " << tests.sourceFilePath <<
+                    " tried to generate tests for method " << testMethod.methodName << 
+                    " from another source file: " << testMethod.sourceFilePath << '.' <<
+                    " This can cause invalid generation.\n";
+        LOG_S(WARNING) << message.str();
+    }
+
```

Comment:
Try to use StringUtils::stringFormat. It is the same as python format string. For [example](https://github.com/UnitTestBot/UTBotCpp/blob/6f9a1e0953672dc84b6642a4262318917dfbdf11/server/src/KleeRunner.cpp#L201).